index.html

```diff
@@ -1388,16 +1388,17 @@ <h2>
       </pre>
 
       <p class="note" title="Determining the DID controller">
-If the <code><a>controller</a></code> property is not present, the
-<a>DID controller</a> can be determined by consulting the respective
-<a>DID method</a> specification. It cannot be assumed that the
-<a>DID subject</a> is also the controller of its own <a>DID</a>, nor
-can it be assumed that <em>only</em> entities identified by <a>DIDs</a>
-present in the value of <code><a>controller</a></code> are capable of exerting
-control over the <a>DID</a> in question. Similarly, <a>DID methods</a> are not
-obliged to respect the value of <code><a>controller</a></code> and can specify
-means for updating the <a>DID document</a> that are unrelated to the
-<code><a>controller</a></code> property.
+If the <code><a>controller</a></code> property is present, its value expresses
+the root of authority for information expressed in the <a>DID Document.</a> If
+the <code><a>controller</a></code> property is not present, the value can be
+inferred to be the same as that of the <code><a>id</a></code> property of the
+<a>DID document</a>. The way to determine how the <a>DID Document</a> can be
+changed is by consulting the relevant <a>DID method</a>. If no <a>verification
+methods</a> are present in the <a>DID Document</a>, then a
+<code><a>controller</a></code> property MUST be present, otherwise the
+the <a>DID document</a> can no longer be changed beyond whatever its present
+value is.
```

Comment:
@jandrieu and @brentzundel -- what about this? I'm trying to cover all of the concerns here at once: ensuring we're speaking clearly to what `controller` enables verifiers to *consider* in a DID-method-agnostic way and indicating that DID methods may say whatever they want about what a verifier actually needs to accept for a DID Document update.

```suggestion
If the <code><a>controller</a></code> property is present, its value expresses
the root of authority for information expressed in the <a>DID Document</a>.
Expressing a <code><a>controller</a></code> property in a <a>DID Document</a>
enables <a>verifiers</a> to consider proofs generated by a
<a>DID controller</a> on behalf of a <a>DID subject</a>, even the same
<a>DID Document</a> expresses no <a>verification methods</a>. Note that
whether any proofs will ultimately be accepted for some purpose is within the
purview of the <a>verifier</a>. DID methods, for example, may require
<a>verifiers</a> to use a mechanism totally unrelated to <a>verification
methods</a> for validating DID Document updates.
```

Reply:
This text is fine for me, except the last sentence. I keep trying to parse it and failing, so I'm not sure sure you're trying to say there.